library/axi_clkgen/axi_clkgen_ip.tcl

```diff
@@ -41,6 +41,22 @@ set_property -dict [list \
 	widget {checkBox} \
 ] $param
 
+set_property value_format string [ipx::get_hdl_parameters SPEED_GRADE -of_objects [ipx::current_core]]
+set_property value_format string [ipx::get_hdl_parameters SPEED_GRADE -of_objects [ipx::current_core]]
+
+set part [get_property PART [current_project]]
+foreach x [list_property $part] {
+  if { $x == "SPEED_GRADE" } {
```

Comment:
I created a sample project (zcu102) and "SPEED_GRADE" was not listed in the properties of the part, but "SPEED" was.

Reply:
I made a find and replace change. Thanks